foxy/ci-nightly-debug.yaml

```diff
@@ -28,7 +28,7 @@ jenkins_job_label: ci-agent
 jenkins_job_priority: 47
 jenkins_job_schedule: 15 23 * * *
 jenkins_job_timeout: 300
-package_selection_args: '--packages-ignore rmw_fastrtps_dynamic_cpp'
+package_selection_args: '--packages-ignore rmw_fastrtps_dynamic_cpp test_communication'
```

Comment:
I'll just point out that this change removes test coverage of single rmw communication tests for Debug and Release. Though perhaps this is something we can live without, considering the tests exist for build type "None".

Reply:
Actually I think the new cross vendor builds will not only test A<->B but also A<->A and B<->B since `test_communication always tests all available combinations.

Maybe we can narrow that down in a future PR.

Reply:
Another avenue would be to override the cross vendor tests at the package level. E.g. pass a cmake argument to `test_communication` instead of the function to get rmws in ros2/rmw#199.

Reply:
That would be possible. Filtering the available RMW implementations has more use cases though: you can install Debian packages or a packaging archive supporting multiple RMW and then build an overlay workspace restricted to only a subset / one RMW impl.

Reply:
I will follow up with a PR for `test_communication` to skip X<->X tests somehow - either when `RMW_IMPLEMENTATIONS` is set or another new option.

Reply:
Sounds good.

Reply:
See ros2/system_tests#403 and 2775357bf8545a1ce32d0a7cd766417acb666abe.

Reply:
Now that we're skipping multi-rmw tests in `test_communication` directly, I don't think we need to ignore the package when building anymore.